designsafe/apps/projects/managers/publication.py

```diff
@@ -196,15 +196,16 @@ def publish_resource(project_id, entity_uuids=None, publish_dois=False):
     responses = []
 
     if publish_dois:
```

Comment:
It looks like we can remove an indent level by doing `if publish_dois and entity_uuids:`